connectors/kafka/src/schemas/Test.avsc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
The avro-generated code is still appearing in the packaged jar; maybe you can get rid of it by making sbt `clean` before `publishLocal` and then making `avro`depend on `test:compile` instead of `compile`.